mainapp/models.py

```diff
@@ -3,61 +3,58 @@
 from django.core.validators import MinLengthValidator, int_list_validator
 from datetime import datetime
 
+
 # Create your models here.
 
 class Comment(models.Model):
-    id = models.BigAutoField('id',primary_key=True)
+    id = models.BigAutoField('id', primary_key=True)
     content = models.TextField('content', max_length=400, blank=False)
     created_at = models.DateTimeField('created_at', default=datetime.utcnow)
     deleted_at = models.DateTimeField('deleted_at', null=True)
-    user_id = models.ForeignKey('User_agent', on_delete=models.SET_NULL, null=True)
+    user_id = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
```

Comment:
1. `models.ForeignKey` (many-to-one). Single user would have none or many comments.
2. Must relate to `Profile` instead of `AUTH_USER_MODEL`.